var/sabath/assets/sabath/datasets/n/nanoconfinement.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
For easier reviews, it may be useful to submit a single data set in a single PR.